protobuf-lite/src/main/java/io/grpc/protobuf/lite/ProtoLiteUtils.java

```diff
@@ -38,13 +38,16 @@
 import com.google.protobuf.InvalidProtocolBufferException;
 import com.google.protobuf.MessageLite;
 import com.google.protobuf.Parser;
+import com.google.protobuf.UnsafeByteOperations;
 
 import io.grpc.ExperimentalApi;
 import io.grpc.Metadata;
 import io.grpc.MethodDescriptor.Marshaller;
 import io.grpc.Status;
+import io.grpc.internal.ReadableBuffer;
```

Comment:
Right now InputStream is a common interface that internal and protolite both have.  One of them needs to depend on the other to in order to cast the InputStream to a shared interface.  (and somehow turn it into a byte array / byte buffer.

Other interfaces in mind?